avionics/envs/x86/lib/HAL/time.cpp

```diff
@@ -35,11 +30,15 @@ void initialSystem() {
         std::chrono::time_point_cast<ms>(std::chrono::steady_clock::now());
 }
 
+/**
+ * @brief Retuns the current time in milliseconds relative to startup_t_point.
+ */
 t_point now_ms() {
```

Comment:
@jesseli2002 I'd appreciate it if you could give this function a good look over: it seems to work well during bench testing but still confuses me somewhat.

Reply:
Seems fine.

Reply:
Probably best to put in a comment noting that passing 0 ends up having no effect on the resulting time, though - it's too clever to figure out if you're tired. Maybe slip in a note in the confluence spec as well.